holoviews/ipython/magics.py

```diff
@@ -364,74 +394,83 @@ def update_options(cls, options, items):
         Switch default options and backend if new backend
         is supplied in items.
         """
-        backend = items.get('backend', '')
+        # Get previous backend
         prev_backend = Store.current_backend
-        renderer = Store.renderers[Store.current_backend]
+        renderer = Store.renderers[prev_backend]
```

Comment:
Hmm, so you just want to switch ``Get new backend`` and ``Get previous backend`` blocks around?